src/SipHash.chpl

```diff
@@ -104,6 +128,31 @@ module SipHash {
     return computeSipHash(msg, D, outlen);
   }
   
+  private proc computeSipHashLocalized(msg: [] int, D, param outlen: int) {
```

Comment:
It's unfortunate the proc needed to be copied just to accept a different type arg.  I wonder if the typing was removed if Chapel would be happy?

Reply:
It worked! Good call

Reply:
Note that you should be able to use `integral` to cover both `int` and `uint(8)` without going totally generic.

Alternatively if you want to just restrict to int/uint(8) you should be able to do something like:

```chpl
private proc computeSipHashLocalized(msg: [] ?t, D, param outlen: int) where t == int || t == uint(8) {
```

Reply:
@ronawho  Oh.. that's cool!